src/CreateRecordsWrapper.js

```diff
@@ -43,10 +43,35 @@ const CreateRecordsWrapper = ({
     createItemRecord,
   },
 }) => {
-  const { identifierTypesByName } = useData();
+  const { identifierTypesByName, instanceStatuses, settings } = useData();
   const callout = useCallout();
   const isLoading = useIsLoading();
 
+  const value = settings[0]?.value || '';
+
+  let config;
+
+  try {
+    const { instanceStatusCode, defaultDiscoverySuppress } = JSON.parse(value);
+
+    // Since the Checkbox implicitly casts any non-empty string to true,
+    // had to explicitly convert the 'false'/'true' string to the boolean.
+    const discoverySuppress = JSON.parse(defaultDiscoverySuppress);
+    const statusId = (instanceStatuses.find(status => status.code === instanceStatusCode) || {}).id || '';
+
+    const instance = {
```

Comment:
It looks like the `instance` const (the name) here collides with the `instance` in `handleSubmit`. Would it be possible to rename one of them?